extension/package.json

```diff
@@ -30,7 +30,7 @@
     "@toruslabs/openlogin-jrpc": "^1.3.3",
     "advanced-css-reset": "^1.2.2",
     "async-mutex": "^0.3.2",
-    "bls-wallet-clients": "0.2.0",
+    "bls-wallet-clients": "0.5.2",
```

Comment:
This should now be `0.5.3` in `main`, this diff will be removed after @chaitanyapotti updates `feat/inpage-provider` with `main`.

Reply:
resolved in f30cd85